wpilibj/src/athena/java/edu/wpi/first/wpilibj/AnalogPotentiometer.java

```diff
@@ -54,8 +41,11 @@ private void initPot(final AnalogInput input, double fullRange, double offset) {
    */
   public AnalogPotentiometer(final int channel, double fullRange, double offset) {
```

Comment:
Can this be a delegating constructor?  e.g. this(new AnalogInput(channel), fullRange, offset)?